airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -46,14 +46,13 @@ class WasbTaskHandler(FileTaskHandler, LoggingMixin):
 
     def __init__(
         self,
+        *,
```

Comment:
I think moving `*` to this place is a breaking change.

Reply:
I have removed. It's not needed anymore.